code/game/machinery/doors/windowdoor.dm

```diff
@@ -65,12 +65,8 @@
 	AddElement(/datum/element/atmos_sensitive, mapload)
 
 /obj/machinery/door/window/Destroy()
-	set_density(FALSE)
```

Comment:
@LemonInTheDark can correct me but i believe that this set_density() call is necessary so air_update_turf() updates geometry correctly.

Reply:
this is correct, density setting can be thought of as a cleanup operation rather then a side effect @Profakos, it should always happen when an object is deleted so it should happen on destroy

Reply:
I see, I did not know this! I shall fix this posthaste.